pkg/services/accesscontrol/ossaccesscontrol/ossaccesscontrol.go

```diff
@@ -57,3 +58,15 @@ func (ac *OSSAccessControlService) GetUserPermissions(ctx context.Context, user
 
 	return permissions, nil
 }
+
+func (ac *OSSAccessControlService) GetUserBuiltInRoles(user *models.SignedInUser) []string {
```

Comment:
Nit, but I would just inline this method since there's no benefit to having it as a method, plus it being an exported method it fattens the `OSSAccessControlService` interface.

Reply:
The only reason to have it as a method - I call it from the Enterprise part (as a fallback if there's no license). So I have to take it out from `GetUserPermissions()`.

Reply:
It's not necessary for this function to be a method of `OSSAccessControlService`, but it should be exported at least. So having it as a method looks a bit more consistent.

Reply:
@alexanderzobnin ah thanks for explaining that! I never think of that possibility, i e. things being exported for the sake of enterprise.